Formula/bash.rb

```diff
@@ -6,7 +6,7 @@ class Bash < Formula
   stable do
     url "https://ftp.gnu.org/gnu/bash/bash-5.0.tar.gz"
     mirror "https://mirrors.ocf.berkeley.edu/gnu/bash/bash-5.0.tar.gz"
```

Comment:
The check also complains about this mirror URL not being `deb.debian.org`, but it would be a duplicate of the following line if I did so. Should I remove it, or are multiple `mirror` stanzas that fail `brew audit` OK?

Reply:
Remove it, fix audit.